src/lfx/README.md

```
@@ -1,211 +1,405 @@
+### LFX serve options
+
+| Option | Description |
+|--------|--------------|
+| `--check-variables` / `--no-check-variables` | Check global variables for environment compatibility. Default: `--check-variables`. |
+| `--env-file` | Path to the `.env` file containing environment variables. |
+| `--host`, `-h` | Host to bind the server to. Default: `127.0.0.1`. |
+| `--log-level` | Logging level. One of: `debug`, `info`, `warning`, `error`, `critical`. Default: `warning`. |
+| `--port`, `-p` | Port to bind the server to. Default: `8000`. |
+| `--verbose`, `-v` | Show diagnostic output and execution details. |
+| `--flow-json` | Read inline flow JSON content as a string. Example: `uv run lfx serve --flow-json '{...}'`. |
+| `--stdin` | Read JSON flow content from `stdin`. Example: `cat flow.json | uv run lfx serve --stdin`. |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)